t/lib/OpenQA/Test/Utils.pm

```diff
@@ -369,6 +369,12 @@ sub job_create {
     return $job;
 }
 
+sub find_most_recent_event {
```

Comment:
This looks like a left over since you have the same function already in `t/lib/OpenQA/Test/Case.pm`. (I actually don't care whether it is in `t/lib/OpenQA/Test/Case.pm` or `t/lib/OpenQA/Test/Utils.pm` but pick only one of them.)

Reply:
I didn't realize I put it in Utils as well, thanks for catching that!